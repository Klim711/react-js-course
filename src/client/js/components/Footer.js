import React, {PureComponent} from 'react';

class Footer extends PureComponent {
  render() {
    return (
      <footer className="row page-footer">
        <div className="title">netflixroulette</div>
      </footer>
    );
  }
}

export default Footer;
