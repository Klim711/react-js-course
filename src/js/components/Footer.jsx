import React, {PureComponent} from 'react';

class Footer extends PureComponent {
  render() {
    return <footer>{this.props.content}</footer>;
  }
}

export default Footer;
