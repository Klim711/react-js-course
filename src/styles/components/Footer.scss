@import '../constants.scss';

.page-footer {
    background-color: $footer-color;
}

.row.page-footer {
    padding-top: 20px;
    padding-bottom: 20px;
}