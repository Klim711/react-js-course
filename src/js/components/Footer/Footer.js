import React, {PureComponent} from 'react';
import './Footer.scss';

class Footer extends PureComponent {
  render() {
    return (
      <footer className="row page-footer">
        <div className="title">netflixroulette</div>
      </footer>
    );
  }
}

export default Footer;
