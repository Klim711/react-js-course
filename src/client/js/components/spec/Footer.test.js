import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../Footer';

describe('Footer component', () => {
  it('matches to snapshot', () => {
    const renderedValue = renderer.create(<Footer/>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
