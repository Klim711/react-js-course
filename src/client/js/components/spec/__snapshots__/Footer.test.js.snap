// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches to snapshot 1`] = `
<footer
  className="row page-footer"
>
  <div
    className="title"
  >
    netflixroulette
  </div>
</footer>
`;
